src/hooks/useConfig.tsx

```diff
@@ -1,9 +1,10 @@
 // external
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
why dont we just add the preact eslint rules: https://github.com/transcend-io/main/blob/4eaf2ddc28d95fa5077f61f8e3199d3355caf2e9/.eslintrc.js#L58